infrastructure/main.tf

```diff
@@ -27,6 +27,12 @@ locals {
 
 }
 
+data "azurerm_subnet" "postgres" {
+  name                 = "core-infra-subnet-0-${var.env}"
+  resource_group_name  = "core-infra-${var.env}"
+  virtual_network_name = "core-infra-vnet-${var.env}"
+}
+
 module "repl-docmosis-backend" {
```

Comment:
I don't think this is required anymore. The repo has been archived
https://github.com/hmcts/cnp-module-webapp
Could you check with platops if it can safely be removed